cmd/nerdctl/ipfs/ipfs_simple_linux_test.go

```diff
@@ -142,11 +142,11 @@ func TestIPFSSimple(t *testing.T) {
 				helpers.Ensure("pull", "ipfs://"+data.Get(transformedImageCIDKey))
 			},
 			Cleanup: func(data test.Data, helpers test.Helpers) {
-				helpers.Anyhow("rm", data.Identifier("commit-container"))
-				helpers.Anyhow("rmi", data.Identifier("commit-image"))
+				helpers.Anyhow("rm", "-f", data.Identifier("commit-container"))
+				helpers.Anyhow("rmi", "-f", data.Identifier("commit-image"))
 				if data.Get(mainImageCIDKey) != "" {
-					helpers.Anyhow("rmi", data.Get(mainImageCIDKey))
-					helpers.Anyhow("rmi", data.Get(transformedImageCIDKey))
+					helpers.Anyhow("rmi", "-f", data.Get(mainImageCIDKey))
+					helpers.Anyhow("rmi", "-f", data.Get(transformedImageCIDKey))
```

Comment:
Can you split the rmi -f part to a separate PR?

Reply:
Done.